### ISSUE TITLE
de/increased circulating factor XI level

### ISSUE THREAD

**Issue by slaulederkind** [mgijax/mammalian-phenotype-ontology#3356]
>If this is a request for a new term, please provide us with the following information.

>1. Preferred term label (e.g., asplenia)

1. abnormal circulating factor XI level
2. decreased circulating factor XI level
3. increased circulating factor XI level

>2. Synonyms (e.g., absent spleen)

2. reduced circulating factor XI level
3. elevated circulating factor XI level

>3. Textual definition (the definition should be understandable even for non-specialists. Include a PubMed ID or URL(s) to refer to any relevant article(s) that provides information about the suggested term).

1.  aberrant concentration in the blood of plasma thromboplastin antecedent, the zymogen form of factor XIa, one of the serine protease of the coagulation cascade.
2. decreased concentration in the blood of plasma thromboplastin antecedent, the zymogen form of factor XIa, one of the serine protease of the coagulation cascade.
3. increased concentration in the blood of plasma thromboplastin antecedent, the zymogen form of factor XIa, one of the serine protease of the coagulation cascade.

>4. Any additional notes to clarify the meaning or use of the term. Please provide a PubMed ID or URL(s) for the source of this information.

PMID:1169825

>5. Suggested parent term (Please look in the hierarchy of the MP using the [MP Browser](http://www.informatics.jax.org/vocab/mp_ontology) or other browsers such as [OLS](http://www.ebi.ac.uk/ols/ontologies/mp) and find the place in the MP where the new term best fits)

1. abnormal circulating enzyme level (MP:0001570)
2.  abnormal circulating factor XI level (new term)
3. abnormal circulating factor XI level (new term)

>6. If you would like a nanoattribution, please indicate your ORCID id or provide a label such as MGI:sbello (organization, first initial-last name)

ORCID:0000-0001-5356-4174

**Comment by sbello** [comment]
abnormal - MP:0020983
increased - MP:0020985
decreased - MP:0020984
